### ISSUE TITLE
Soaking pots can only output up to 64 and any more will only give 64

### ISSUE THREAD

**Issue by iEatShadows** [Heaveness/Rebirth-of-the-Night-BPG-Edition--Future-Updates-#4]
**Describe the bug**
when trying to get collect refractory material from a soaking pot, I noticed I can get 80 from 16 limestone rocks, after collection I only got 64 instead of 80.

**To Reproduce**  

1. Place one soaking pot and will it up to the max.
2. Place the limestone rocks into the pot.
3. Let it soak.
4. Then check the output result. (should say 80 refractory material.)
5. Collect the refractory material. (Should receive only 64.)

**Expected behavior**
   - I would to expect it give me 80 since JEI also says so as well. 1 limestone rock = 5 refractory materials so 16*5 is 80

**Modpack Version:**
   - Public Alpha 1.8

**Server or Singleplayer:**
   - Lan
  
**Version the world was created on:**
   - Public Alpha 1.8

**Comment by iEatShadows** [comment]
https://github.com/Rebirth-of-the-Night/Rebirth-Of-The-Night/issues/812

Something similar

**Comment by Heaveness** [comment]
Yeah this one is a bit odd, but it looks like it is fixed in a more later version of pyrotech. BPG's current version is very outdated now, but I do not have current plans to update it since it's being changed with RotN 3.0. You will just have to wait until the BPG port for that version.
